docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,154 @@
+# How to Deploy to ProdStack 6 Cloud
+
+## Prerequisites
+
+Before deploying the Identity Platform Juju Bundle Terraform module to ProdStack
+6 Cloud, you need to prepare the followings:
+
+- A service environment has been provisioned in the ProdStack 6 Cloud. If not,
+  please refer to [this section](#how-to-deploy-a-service-environment).
+- Submit [a request to IS for SSH key update](https://portal.admin.canonical.com/requests/new)
+and invite two team member to approve it.
+- Set up and connect to the
+  company-wide [VPN](https://wiki.canonical.com/InformationInfrastructure/IS/HowTo/CompanyOpenVPN).
+
+## How to deploy the module
+
+### Deploy the module
+
+IS team provisioned a bastion server for Identity team to use for accessing the
+service environment.
+You can use the generated ssh private key to log into the server by running the
+following command:
+
+```shell
+$ ssh -i <path-to-private-key> <username>@identity-bastion-ps6.internal
+```
+
+**Note:** the username should be your launchpad id.
+
+Once you logged in, list all the service environments created using the
+following command:
+
+```shell
+$ pe -n
+
+sudo -iu stg-iam-bundle se stg-iam-bundle # (0) [LIVE][PS6][K8S] Testing environment for IAM bundle
+```
+
+Run the following to switch to the target service environment:
+
+```shell
+$ pe stg-iam-bundle
+
+This model is managed by Terraform. Please don't make manual changes using
```

Comment:
Is this text the output of the command pe stg-iam-bundle? If so, I'm not sure if it should be a part of the doc, or at least not part of the shell section.

Reply:
Yes, that is the output of the `pe` command. I use `$` prompt in front of any commands to differentiate commands from the outputs.